scripts/generator.py

```diff
@@ -48,6 +55,8 @@ def argument_parser():
                         help='generate intermediary files only')
     parser.add_argument('--include', action='store',
                         help='include user specified directory of custom field definitions')
+    parser.add_argument('--subset', action='store',
+                        help='render a subset of the schema')
```

Comment:
I noticed I hadn't documented the usage of `--include` yet, so I won't ask you to add documentation for `--subset`, we'll do that as a separate PR. I opened #746 for this.

But just to make sure I understand the usage correctly, is it correct to say that given my custom fields in `my-fields.yml` and my subset selectors at `my-subset.yml`, the command would be the following?

`scripts/generator.py --include my-fields.yml --subset my-subset.yml`

Also, this supports passing multiple subsets as well? I'm not 100% sure about the Python args parser, how would this look at the command line?

Is it a repeatable argument (`--subset my-subset.yml  --subset my-other-subset.yml`) or an argument that takes multiple values (`--subset my-subset.yml my-other-subset.yml`)?

Reply:
For the combination of --include and --subset that's the right command to use.

Good point, multiple value support is an important part that I missed.  I just pushed another commit up that enables the second style you mentioned (`--subset my-subset.yml my-other-subset.yml`).  It can be combined with wildcards too, i.e. `--subset dir1/*.yml dir2/*.yml` would get all the subsets from both dir1 and dir2.

Reply:
Nice. And for the record, the same is true of `--include` already